src/datasets/features/image.py

```diff
@@ -167,10 +168,11 @@ def decode_example(self, value: dict, token_per_repo_id=None) -> "PIL.Image.Imag
                     source_url = path.split("::")[-1]
                     try:
                         repo_id = string_to_dict(source_url, config.HUB_DATASETS_URL)["repo_id"]
-                        token = token_per_repo_id.get(repo_id)
+                        token_per_repo_id.get(repo_id)
                     except ValueError:
-                        token = None
-                    with xopen(path, "rb", token=token) as f:
+                        use_auth_token = None
```

Comment:
same here

Reply:
Sorry, I meant that this maybe needed to be reverted as in my previous comment

> you should revert this no ? we need to get the token from token_per_repo_id

where you replied

> no need, I can change the setting of DownloadConfig.